#include "secondwindow.h"
#include "ui_secondwindow.h"
#include <QLineEdit>
#include <stdlib.h>
#include <QString>

SecondWindow::SecondWindow(QWidget *parent) :
    QDialog(parent),
    ui(new Ui::SecondWindow)
{
    ui->setupUi(this);
    ui->label_12->setVisible(false);
    ui->label_13->setVisible(false);
    ui->label_14->setVisible(false);
    ui->label_15->setVisible(false);
    ui->label_16->setVisible(false);
    ui->label_17->setVisible(false);
    ui->label_18->setVisible(false);
    ui->label_19->setVisible(false);
    ui->label_20->setVisible(false);
    ui->label_21->setVisible(false);
    ui->label_22->setVisible(false);
    ui->label_23->setVisible(false);
    ui->label_24->setVisible(false);
    ui->label_25->setVisible(false);
    ui->label_26->setVisible(false);
    ui->label_27->setVisible(false);
    ui->lineEdit_2->setVisible(false);
    ui->lineEdit_3->setVisible(false);
    ui->lineEdit_4->setVisible(false);
    ui->lineEdit_5->setVisible(false);
    ui->lineEdit_6->setVisible(false);
    ui->lineEdit_7->setVisible(false);
    ui->lineEdit_8->setVisible(false);
    ui->lineEdit_9->setVisible(false);
    ui->lineEdit_10->setVisible(false);
    ui->lineEdit_11->setVisible(false);
    ui->lineEdit_12->setVisible(false);
    ui->lineEdit_13->setVisible(false);
    ui->lineEdit_14->setVisible(false);
    ui->lineEdit_15->setVisible(false);
    ui->lineEdit_16->setVisible(false);
    ui->lineEdit_17->setVisible(false);
}

SecondWindow::~SecondWindow()
{
    delete ui;
}

void SecondWindow::on_pushButton_clicked()
{
   QString q = ui->lineEdit->text();
   int k = q.toInt();
   WhatIsNumber(k);
}

void SecondWindow::WhatIsNumber(int k){
   switch (k){
   case 1:
       ui->label_12->setVisible(false);
       ui->label_13->setVisible(false);
       ui->label_14->setVisible(false);
       ui->label_15->setVisible(false);
       ui->label_16->setVisible(false);
       ui->label_17->setVisible(false);
       ui->label_18->setVisible(false);
       ui->label_19->setVisible(false);
       ui->label_20->setVisible(false);
       ui->label_21->setVisible(false);
       ui->label_22->setVisible(false);
       ui->label_23->setVisible(false);
       ui->label_24->setVisible(false);
       ui->label_25->setVisible(false);
       ui->label_26->setVisible(true);
       ui->label_27->setVisible(true);
       ui->lineEdit_2->setVisible(true);
       ui->lineEdit_3->setVisible(true);
       ui->lineEdit_4->setVisible(false);
       ui->lineEdit_5->setVisible(false);
       ui->lineEdit_6->setVisible(false);
       ui->lineEdit_7->setVisible(false);
       ui->lineEdit_8->setVisible(false);
       ui->lineEdit_9->setVisible(false);
       ui->lineEdit_10->setVisible(false);
       ui->lineEdit_11->setVisible(false);
       ui->lineEdit_12->setVisible(false);
       ui->lineEdit_13->setVisible(false);
       ui->lineEdit_14->setVisible(false);
       ui->lineEdit_15->setVisible(false);
       ui->lineEdit_16->setVisible(false);
       ui->lineEdit_17->setVisible(false);
       break;
   case 2:
       ui->label_12->setVisible(false);
       ui->label_13->setVisible(false);
       ui->label_14->setVisible(false);
       ui->label_15->setVisible(false);
       ui->label_16->setVisible(false);
       ui->label_17->setVisible(false);
       ui->label_18->setVisible(false);
       ui->label_19->setVisible(false);
       ui->label_20->setVisible(false);
       ui->label_21->setVisible(false);
       ui->label_22->setVisible(false);
       ui->label_23->setVisible(false);
       ui->label_24->setVisible(true);
       ui->label_25->setVisible(true);
       ui->label_26->setVisible(true);
       ui->label_27->setVisible(true);
       ui->lineEdit_2->setVisible(true);
       ui->lineEdit_3->setVisible(true);
       ui->lineEdit_4->setVisible(true);
       ui->lineEdit_5->setVisible(true);
       ui->lineEdit_6->setVisible(false);
       ui->lineEdit_7->setVisible(false);
       ui->lineEdit_8->setVisible(false);
       ui->lineEdit_9->setVisible(false);
       ui->lineEdit_10->setVisible(false);
       ui->lineEdit_11->setVisible(false);
       ui->lineEdit_12->setVisible(false);
       ui->lineEdit_13->setVisible(false);
       ui->lineEdit_14->setVisible(false);
       ui->lineEdit_15->setVisible(false);
       ui->lineEdit_16->setVisible(false);
       ui->lineEdit_17->setVisible(false);
       break;
   case 3:
       ui->label_12->setVisible(false);
       ui->label_13->setVisible(false);
       ui->label_14->setVisible(false);
       ui->label_15->setVisible(false);
       ui->label_16->setVisible(false);
       ui->label_17->setVisible(false);
       ui->label_18->setVisible(false);
       ui->label_19->setVisible(false);
       ui->label_20->setVisible(false);
       ui->label_21->setVisible(false);
       ui->label_22->setVisible(true);
       ui->label_23->setVisible(true);
       ui->label_24->setVisible(true);
       ui->label_25->setVisible(true);
       ui->label_26->setVisible(true);
       ui->label_27->setVisible(true);
       ui->lineEdit_2->setVisible(true);
       ui->lineEdit_3->setVisible(true);
       ui->lineEdit_4->setVisible(true);
       ui->lineEdit_5->setVisible(true);
       ui->lineEdit_6->setVisible(true);
       ui->lineEdit_7->setVisible(true);
       ui->lineEdit_8->setVisible(false);
       ui->lineEdit_9->setVisible(false);
       ui->lineEdit_10->setVisible(false);
       ui->lineEdit_11->setVisible(false);
       ui->lineEdit_12->setVisible(false);
       ui->lineEdit_13->setVisible(false);
       ui->lineEdit_14->setVisible(false);
       ui->lineEdit_15->setVisible(false);
       ui->lineEdit_16->setVisible(false);
       ui->lineEdit_17->setVisible(false);
       break;
   case 4:
       ui->label_12->setVisible(false);
       ui->label_13->setVisible(false);
       ui->label_14->setVisible(false);
       ui->label_15->setVisible(false);
       ui->label_16->setVisible(false);
       ui->label_17->setVisible(false);
       ui->label_18->setVisible(false);
       ui->label_19->setVisible(false);
       ui->label_20->setVisible(true);
       ui->label_21->setVisible(true);
       ui->label_22->setVisible(true);
       ui->label_23->setVisible(true);
       ui->label_24->setVisible(true);
       ui->label_25->setVisible(true);
       ui->label_26->setVisible(true);
       ui->label_27->setVisible(true);
       ui->lineEdit_2->setVisible(true);
       ui->lineEdit_3->setVisible(true);
       ui->lineEdit_4->setVisible(true);
       ui->lineEdit_5->setVisible(true);
       ui->lineEdit_6->setVisible(true);
       ui->lineEdit_7->setVisible(true);
       ui->lineEdit_8->setVisible(true);
       ui->lineEdit_9->setVisible(true);
       ui->lineEdit_10->setVisible(false);
       ui->lineEdit_11->setVisible(false);
       ui->lineEdit_12->setVisible(false);
       ui->lineEdit_13->setVisible(false);
       ui->lineEdit_14->setVisible(false);
       ui->lineEdit_15->setVisible(false);
       ui->lineEdit_16->setVisible(false);
       ui->lineEdit_17->setVisible(false);
       break;
   case 5:
       ui->label_12->setVisible(false);
       ui->label_13->setVisible(false);
       ui->label_14->setVisible(false);
       ui->label_15->setVisible(false);
       ui->label_16->setVisible(false);
       ui->label_17->setVisible(false);
       ui->label_18->setVisible(true);
       ui->label_19->setVisible(true);
       ui->label_20->setVisible(true);
       ui->label_21->setVisible(true);
       ui->label_22->setVisible(true);
       ui->label_23->setVisible(true);
       ui->label_24->setVisible(true);
       ui->label_25->setVisible(true);
       ui->label_26->setVisible(true);
       ui->label_27->setVisible(true);
       ui->lineEdit_2->setVisible(true);
       ui->lineEdit_3->setVisible(true);
       ui->lineEdit_4->setVisible(true);
       ui->lineEdit_5->setVisible(true);
       ui->lineEdit_6->setVisible(true);
       ui->lineEdit_7->setVisible(true);
       ui->lineEdit_8->setVisible(true);
       ui->lineEdit_9->setVisible(true);
       ui->lineEdit_10->setVisible(true);
       ui->lineEdit_11->setVisible(true);
       ui->lineEdit_12->setVisible(false);
       ui->lineEdit_13->setVisible(false);
       ui->lineEdit_14->setVisible(false);
       ui->lineEdit_15->setVisible(false);
       ui->lineEdit_16->setVisible(false);
       ui->lineEdit_17->setVisible(false);
       break;
   case 6:
       ui->label_12->setVisible(false);
       ui->label_13->setVisible(false);
       ui->label_14->setVisible(false);
       ui->label_15->setVisible(false);
       ui->label_16->setVisible(true);
       ui->label_17->setVisible(true);
       ui->label_18->setVisible(true);
       ui->label_19->setVisible(true);
       ui->label_20->setVisible(true);
       ui->label_21->setVisible(true);
       ui->label_22->setVisible(true);
       ui->label_23->setVisible(true);
       ui->label_24->setVisible(true);
       ui->label_25->setVisible(true);
       ui->label_26->setVisible(true);
       ui->label_27->setVisible(true);
       ui->lineEdit_2->setVisible(true);
       ui->lineEdit_3->setVisible(true);
       ui->lineEdit_4->setVisible(true);
       ui->lineEdit_5->setVisible(true);
       ui->lineEdit_6->setVisible(true);
       ui->lineEdit_7->setVisible(true);
       ui->lineEdit_8->setVisible(true);
       ui->lineEdit_9->setVisible(true);
       ui->lineEdit_10->setVisible(true);
       ui->lineEdit_11->setVisible(true);
       ui->lineEdit_12->setVisible(true);
       ui->lineEdit_13->setVisible(true);
       ui->lineEdit_14->setVisible(false);
       ui->lineEdit_15->setVisible(false);
       ui->lineEdit_16->setVisible(false);
       ui->lineEdit_17->setVisible(false);
       break;
   case 7:
       ui->label_12->setVisible(false);
       ui->label_13->setVisible(false);
       ui->label_14->setVisible(true);
       ui->label_15->setVisible(true);
       ui->label_16->setVisible(true);
       ui->label_17->setVisible(true);
       ui->label_18->setVisible(true);
       ui->label_19->setVisible(true);
       ui->label_20->setVisible(true);
       ui->label_21->setVisible(true);
       ui->label_22->setVisible(true);
       ui->label_23->setVisible(true);
       ui->label_24->setVisible(true);
       ui->label_25->setVisible(true);
       ui->label_26->setVisible(true);
       ui->label_27->setVisible(true);
       ui->lineEdit_2->setVisible(true);
       ui->lineEdit_3->setVisible(true);
       ui->lineEdit_4->setVisible(true);
       ui->lineEdit_5->setVisible(true);
       ui->lineEdit_6->setVisible(true);
       ui->lineEdit_7->setVisible(true);
       ui->lineEdit_8->setVisible(true);
       ui->lineEdit_9->setVisible(true);
       ui->lineEdit_10->setVisible(true);
       ui->lineEdit_11->setVisible(true);
       ui->lineEdit_12->setVisible(true);
       ui->lineEdit_13->setVisible(true);
       ui->lineEdit_14->setVisible(true);
       ui->lineEdit_15->setVisible(true);
       ui->lineEdit_16->setVisible(false);
       ui->lineEdit_17->setVisible(false);
       break;
   case 8:
       ui->label_12->setVisible(true);
       ui->label_13->setVisible(true);
       ui->label_14->setVisible(true);
       ui->label_15->setVisible(true);
       ui->label_16->setVisible(true);
       ui->label_17->setVisible(true);
       ui->label_18->setVisible(true);
       ui->label_19->setVisible(true);
       ui->label_20->setVisible(true);
       ui->label_21->setVisible(true);
       ui->label_22->setVisible(true);
       ui->label_23->setVisible(true);
       ui->label_24->setVisible(true);
       ui->label_25->setVisible(true);
       ui->label_26->setVisible(true);
       ui->label_27->setVisible(true);
       ui->lineEdit_2->setVisible(true);
       ui->lineEdit_3->setVisible(true);
       ui->lineEdit_4->setVisible(true);
       ui->lineEdit_5->setVisible(true);
       ui->lineEdit_6->setVisible(true);
       ui->lineEdit_7->setVisible(true);
       ui->lineEdit_8->setVisible(true);
       ui->lineEdit_9->setVisible(true);
       ui->lineEdit_10->setVisible(true);
       ui->lineEdit_11->setVisible(true);
       ui->lineEdit_12->setVisible(true);
       ui->lineEdit_13->setVisible(true);
       ui->lineEdit_14->setVisible(true);
       ui->lineEdit_15->setVisible(true);
       ui->lineEdit_16->setVisible(true);
       ui->lineEdit_17->setVisible(true);
       break;
    default:
       return ui->label_3->setText("Введите от 1 до 8");
   }
}
